auth/src/main/java/com/alibaba/nacos/auth/filter/AuthJwtAuthenticationTokenFilter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 1999-2021 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.auth.filter;
+
+import com.alibaba.nacos.api.common.Constants;
+import com.alibaba.nacos.auth.AuthJwtTokenManager;
+import com.alibaba.nacos.auth.AuthNacosAuthConfig;
+import com.alibaba.nacos.common.utils.StringUtils;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * jwt auth token filter.
+ *
+ * @author wfnuser
+ */
+public class AuthJwtAuthenticationTokenFilter extends OncePerRequestFilter {
```

Comment:
Whether conflict with `AuthFilter` in Nacos?

Reply:
I have test it and no conflict happen

Reply:
So  what's the sort of the `AuthJwtAuthenticationTokenFilter` and `AuthFilter`?
AuthFilter first or AuthJwtAuthenticationTokenFilter first?